.github/workflows/before_merge.yaml

```diff
@@ -108,21 +108,10 @@ jobs:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/common_repo_setup
 
-      - name: Download Metrics Artifacts for Generate Input Variations Tests
-        uses: actions/download-artifact@v4
-        with:
-          pattern: model-tests-metrics-group-${{ matrix.group }}
-          path: metrics/
       - name: Run Model Input Variations Tests
         run: |
           set +e
-          rm -rf tests/autogen-op/
-          python3 tools/generate_input_variation_test_from_models.py
-          exit_code=$?  # Capture the exit code
-          if [ $exit_code -ne 0 ]; then
-            echo "Failed to generate input variation test : exit code $exit_code.";
-            exit 1;  # Fail the workflow for other errors
-          fi
+          rm -rf tests/autogen-op/ALL
```

Comment:
Why?

Reply:
The `ALL` is the merged of all models, that is, if aten.add.Tensor have input_var_A in modelA and inuput_var_B in modelB, then the aten.add.Tensor test in `ALL` will both contain input_var_A and input_var_B

`python3 -m pytest --github-report tests/autogen-op -s` will run all the test include `ALL` and models, so I remove `tests/autogen-op/ALL` is for avoid running duplicate case.